tests/main/classic-ubuntu-core-transition/task.yaml

```diff
@@ -7,9 +7,6 @@ systems: [-ubuntu-core-16-*, -ubuntu-*-ppc64el]
 warn-timeout: 1m
 kill-timeout: 5m
 
-restore: |
-    rm -f prevValue state.json.copy
```

Comment:
you might want to keep  rm -f state.json.new to be on the safe side if the test breaks in the middle